### PR TITLE
tailcfg: add suggest exit node related node attribute

### PR DESCRIPTION
**Pull request by clairew**
Updates tailscale/corp#17516

### CODE REVIEW

**Review by sailorfrag — APPROVED** [review]

File: tailcfg/tailcfg.go [hunk 1]
```diff
@@ -2214,6 +2214,9 @@ const (
 
 	// NodeAttrsTailFSAccess enables accessing shares via TailFS.
 	NodeAttrsTailFSAccess NodeCapability = "tailfs:access"
+
+	// NodeAttrsSuggestExitNode labels an exit node suggestion candidate.
+	NodeAttrsSuggestExitNode NodeCapability = "suggest-exit-node"
```
Comment (bradfitz) [inline tailcfg/tailcfg.go:2219]: So at most one node has this label?

**Review by bradfitz — APPROVED** [review]
> Updates https://github.com/tailscale/corp/issues/17516

That's an incredibly sparse bug. (no body even)

Can you flesh it out with a link to the design doc(s) at least?

File: tailcfg/tailcfg.go (hunk 1 above)
Comment (clairew) [reply tailcfg/tailcfg.go:2219]: No - we will tag each potential exit node candidate this node attribute as a way for the client to know of control's recommendations

Comment (bradfitz) [reply tailcfg/tailcfg.go:2219]: Let's reword this then. Maybe something like: ?

```
// NodeAttrsSuggestExitNode is applied to each exit node which the control plane has determined
// is a recommended exit node.
```

**Pull request merged**